app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
```

Comment:
Typographical Error: The attribute name `comfort_class` is correct here, but ensure consistency throughout your code. Previously, it was noted as `comform_class`.